oauth2_http/javatests/com/google/auth/oauth2/ImpersonatedCredentialsTest.java

```diff
@@ -142,6 +158,160 @@ private GoogleCredentials getSourceCredentials() throws IOException {
     return sourceCredentials;
   }
 
+  @Test()
+  public void fromJson_userAsSource_WithQuotaProjectId() throws IOException {
+    GenericJson json =
+        buildImpersonationCredentialsJson(
+            IMPERSONATION_URL,
+            DELEGATES,
+            QUOTA_PROJECT_ID,
+            USER_ACCOUNT_CLIENT_ID,
+            USER_ACCOUNT_CLIENT_SECRET,
+            REFRESH_TOKEN);
+    MockIAMCredentialsServiceTransportFactory mtransportFactory =
```

Comment:
mockTransportFactory because style does not abbreviate in variable names

Reply:
Done.